samples/nrf9160/azure_fota/Kconfig

```diff
@@ -14,4 +14,10 @@ module = AZURE_FOTA_SAMPLE
 module-str = Azure FOTA sample
 source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
 
+config DEVICE_ID_USE_HW_ID
+	bool "Use HW ID as device ID"
+	help
+		Use the devices's hardware ID as device ID when connecting to Azure IOT hub
```

Comment:
Nit: The help text should be indented by two spaces instead of 1 tab (for some strange reason)